vdoctl/cmd/root.go

```diff
@@ -156,3 +162,27 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
 	return nil
 }
+
+func getVdoNamespace() string {
+	// List Deployments
+	ctxNew := context.Background()
+	deploymentList := &appsv1.DeploymentList{}
+	err := K8sClient.List(ctxNew, deploymentList)
+
+	if err != nil {
+		cobra.CheckErr(err)
+	}
+
+	for _, deployment := range deploymentList.Items {
+		if deployment.Name == VdoDeploymentName {
+			VdoCurrentNamespace = deployment.Namespace
+		}
+	}
+	return VdoCurrentNamespace
+}
+
+func checkVdoDeployment() {
```

Comment:
rather than checking vdoDeployment from other part of the code. Lets exit vdoctl with an error saying could not find the namespace for vdo operator after line 127